mu/locale/fr/LC_MESSAGES/mu.po

```diff
@@ -584,579 +74,633 @@ msgstr ""
 "forme:\n"
 "NOM=VALEUR"
 
-#: mu/interface/dialogs.py:150
+#: mu/interface/dialogs.py:183
 msgid "Minify Python code before flashing?"
-msgstr "Compresser le code Python avant le flasher?"
+msgstr "Compresser le code Python avant de le flasher?"
 
-#: mu/interface/dialogs.py:153
+#: mu/interface/dialogs.py:187
 msgid ""
 "Override the built-in MicroPython runtime with the following hex file (empty "
 "means use the default):"
 msgstr ""
-"Remplace l'exécutable MicroPython par le fichier hex suivant (vide signifie "
-"utilise celui par défaut):"
+"Remplace l'exécutable MicroPython par le fichier hex suivant (laisser vide "
+"pour utiliser celui par défaut):"
 
-#: mu/interface/dialogs.py:175 mu/interface/main.py:973
+#: mu/interface/dialogs.py:213
+msgid ""
+"The packages shown below will be available to import in Python 3 mode. "
+"Delete a package from the list to remove its availability.\n"
+"\n"
+"Each separate package name should be on a new line. Packages are installed "
+"from PyPI (see: https://pypi.org/)."
+msgstr ""
+"Les paquets affichés ci-dessous pourront être importés dans le mode Python "
+"3. Efface un paquet de la liste pour enlever sa disponibilité.\n"
+"\n"
+"Chaque paquet doit être sur sa propre ligne. Les paquets sont installés "
+"depuis PyPI (voir: https://pypi.org/)."
+
+#: mu/interface/dialogs.py:239 mu/interface/main.py:1231
 msgid "Mu Administration"
 msgstr "Administration de Mu"
 
-#: mu/interface/dialogs.py:186
+#: mu/interface/dialogs.py:253
 msgid "Current Log"
 msgstr "Log courant"
 
-#: mu/interface/dialogs.py:189
+#: mu/interface/dialogs.py:256
 msgid "Python3 Environment"
-msgstr "Environnement Python 3"
+msgstr "Environnement Python3"
 
-#: mu/interface/dialogs.py:194
+#: mu/interface/dialogs.py:262
 msgid "BBC micro:bit Settings"
 msgstr "Configuration BBC micro:bit"
 
-#: mu/interface/dialogs.py:223
+#: mu/interface/dialogs.py:265
+msgid "Third Party Packages"
+msgstr "Paquets tiers"
+
+#: mu/interface/dialogs.py:295
 msgid "Find / Replace"
 msgstr "Rechercher / Remplacer"
 
-#: mu/interface/dialogs.py:227
+#: mu/interface/dialogs.py:299
 msgid "Find:"
 msgstr "Recherche:"
 
-#: mu/interface/dialogs.py:233
+#: mu/interface/dialogs.py:306
 msgid "Replace (optional):"
 msgstr "Remplace (optionnel):"
 
-#: mu/interface/dialogs.py:239
+#: mu/interface/dialogs.py:312
 msgid "Replace all?"
 msgstr "Remplacer tout?"
 
-#: mu/interface/editor.py:222 mu/logic.py:960
+#: mu/interface/dialogs.py:361
+msgid "Third Party Package Status"
+msgstr "Statut des paquets tiers"
```

Comment:
The original string is singular, but from context this is about all installed packages.